crates/ty_ide/src/completion.rs

```diff
@@ -4216,6 +4252,32 @@ type <CURSOR>
         ");
     }
 
+    #[test]
+    fn from_import_i_suggests_import() {
+        let builder = completion_test_builder("from typing i<CURSOR>");
+
+        let completions_test = builder.build();
+
+        completions_test.contains("import");
+
+        let completions = completions_test.completions();
+
+        assert!(completions.len() == 1);
+    }
+
+    #[test]
+    fn from_import_random_name_suggests_import() {
```

Comment:
Can you add a comment noting that this might not work the same way in real LSP clients (since they might do their own filtering based on what has been typed)?